fastlane/swift/Plugins.swift

```diff
@@ -0,0 +1,11 @@
+// This class is automatically included in FastlaneRunner during build
+
+// This autogenerated file will be overwritten or replaced when installing/updating plugins or running "fastlane generate_swift"
+//
+//  ** NOTE **
+//  This file is provided by fastlane and WILL be overwritten in future updates
+//  If you want to add extra functionality to this project, create a new file in a
+//  new group so that it won't be marked for upgrade
+//
+
+import Foundation
```

Comment:
Thanks a lot for providing Testing Steps 💪 
I was following these:

> Plugin
bundle exec fastlane init swift
bundle exec fastlane add_plugin ascii_art
fastlane/swift/Plugins.swift should have action for asciiArt()
Update fastlane/Fastlane.swift with a call for asciiArt()

however trying to add a different plugin: [`secret_keeper`](https://github.com/mollyIV/fastlane-plugin-secret_keeper). 

![image](https://user-images.githubusercontent.com/10795657/63293866-4a2a1e00-c2c9-11e9-953e-a1955f6bd92e.png)

Both `Actions.swift` and `Plugins.swift` are empty - is there anything that I should first do with the plugin itself? 🤔

I did the manual setup (4) when executing `bundle exec fastlane init swift`.